src/track/beatutils.cpp

```diff
@@ -79,37 +83,77 @@ QMap<int, double> BeatUtils::findTempoChanges(
     return stableTemposByPosition;
 }
 
+void BeatUtils::RemoveSmallArrhythmic(
+        QVector<double>& rawBeats,  int sampleRate, QMap<int, double> &stableTemposByPosition) {
+    // A common problem the analyzer has is to detect arrhythmic regions
+    // of tacks with a constant tempo as in a different unsteady tempo. 
+    // This happens frequently on builds and breaks with heavy effects on edm music.
+    // Since these occurs most on beatless regions we do not want them to be
+    // on a different tempo, because they are still syncable in the true tempo
+    // We arbitraly remove these arrhythmic regions if they are short than 16s
+    auto positionsWithTempoChange = stableTemposByPosition.keys();
+    auto tempoValues = stableTemposByPosition.values();
+    QVector<double> fixedBeats;
```

Comment:
I'm unclear which meaning of "fixed" this is using. Are the beats "fixed" as in corrected, or "fixed" as in unmoved?

Reply:
changed to anchored